server/server/src/lib.rs

```diff
@@ -139,6 +140,10 @@ pub async fn start_server(
             .unwrap();
     }
 
+    // SET LOG CALLBACK FOR WASM FUNCTIONS
+    info!("Setting wasm function log callback..");
+    let _ = set_log_callback(|log| println!("log {:?}", log), "info");
+
```

Comment:
Have added call for logging here which fixes https://github.com/msupply-foundation/open-msupply/issues/5182

I added this here because it was helpful for trouble shooting in this PR, and was a small change.

However when I unwrap, it is still throwing an error from being called multiple times. This was unexpected for me? I wonder if it needs to be elsewhere

Reply:
I think this is the correct place, use info! rather then println